code/game/turfs/closed/walls.dm

```diff
@@ -150,15 +150,15 @@
 	return TRUE
 
 /**
-  *Deals damage back to the hulk's arm.
-  *
-  *When a hulk manages to break a wall using their hulk smash, this deals back damage to the arm used.
-  *This is in its own proc just to be easily overridden by other wall types. Default allows for three
-  *smashed walls per arm. Also, we use CANT_WOUND here because wounds are random. Wounds are applied
-  *by hulk code based on arm damage and checked when we call break_an_arm().
-  *Arguments:
-  **arg1 is the arm to deal damage to.
-  **arg2 is the hulk
+ *Deals damage back to the hulk's arm.
+ *
+ *When a hulk manages to break a wall using their hulk smash, this deals back damage to the arm used.
+ *This is in its own proc just to be easily overridden by other wall types. Default allows for three
+ *smashed walls per arm. Also, we use CANT_WOUND here because wounds are random. Wounds are applied
+ *by hulk code based on arm damage and checked when we call break_an_arm().
+ *Arguments:
+ **arg1 is the arm to deal damage to.
+ **arg2 is the hulk
```

Comment:
```suggestion
 * Deals damage back to the hulk's arm.
 *
 * When a hulk manages to break a wall using their hulk smash, this deals back damage to the arm used.
 * This is in its own proc just to be easily overridden by other wall types. Default allows for three
 * smashed walls per arm. Also, we use CANT_WOUND here because wounds are random. Wounds are applied
 * by hulk code based on arm damage and checked when we call break_an_arm().
 * Arguments:
 * * arg1 is the arm to deal damage to.
 * * arg2 is the hulk
```